### PR TITLE
Fixes #93: Fixed `yii\redis\ActiveRecord::deleteAll()` with condition

### LINKED ISSUE #93: deleteAll with strict_types=1
### What steps will reproduce the problem?
Call deleteAll with "IN" condition
\yii\redis\ActiveRecord::deleteAll(['in','id',[1,2,3]);
### What's expected?
Count of deleted in return.
### What do you get instead?
[yii\base\ErrorException] array_intersect(): Argument #1 is not an array

### Additional info
In \yii\yii2-redis\ActiveQuery.php:355
called $modelClass::isPrimaryKey($this->where[1]) 
isPrimaryKey first argument must be array (by phpdoc)

| Q                | A
| ---------------- | ---
| Yii vesion       |2.0.10
| PHP version      | 7.0.4(strict_types=1)
| Operating system | Ubuntu

### PR DESCRIPTION
**Pull request by samdark**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #93

### CODE REVIEW

File: ActiveQuery.php [hunk 1]
```diff
@@ -352,7 +352,7 @@ protected function executeScript($db, $type, $columnName = null)
         // find by primary key if possible. This is much faster than scanning all records
         if (is_array($this->where) && (
                 !isset($this->where[0]) && $modelClass::isPrimaryKey(array_keys($this->where)) ||
-                isset($this->where[0]) && $this->where[0] === 'in' && $modelClass::isPrimaryKey($this->where[1])
+                isset($this->where[0]) && $this->where[0] === 'in' && $modelClass::isPrimaryKey((array)$this->where[1])
```
Comment (arogachev) [inline ActiveQuery.php]: You missed a space between `(array)` and casting value. Even within this file the other occurences of `(array)` are followed by space, so this is inconsistent and small code style violation.

Comment (samdark) [inline ActiveQuery.php]: Fixed.

File: tests/ActiveRecordTest.php [hunk 2]
```diff
@@ -450,4 +450,13 @@ public function testFindEmptyWith()
...
         $this->assertEquals([], $orders);
     }
+
+    /**
+     * @see https://github.com/yiisoft/yii2-redis/issues/93
+     */
+    public function testDeleteAllWithCondition()
+    {
+        $deletedCount = Order::deleteAll(['in', 'id', [1, 2, 3]]);
+        $this->assertEquals(3, $deletedCount);
```
Comment (arogachev) [inline tests/ActiveRecordTest.php]: Maybe convert this to one liner since it fits in one line?

Comment (samdark) [inline tests/ActiveRecordTest.php]: No. The variable is important for understanding the context in this case.

Comment (arogachev) [inline tests/ActiveRecordTest.php]: @samdark Indeed, it would be not absolutely clear without it. Nevermind then.

**Pull request merged**